bitsail-connectors/connector-redis/src/main/java/com/bytedance/bitsail/connector/redis/sink/RedisWriter.java

```diff
@@ -164,21 +222,17 @@ private void validate(Row record) throws BitSailException {
             String.format("record contains null element in index[%d]", i));
       }
     }
-    if (commandDescription.getJedisCommand() == JedisCommand.ZADD) {
-      parseScoreFromBytes((byte[]) record.getField(1));
-    }
   }
 
   /**
    * check if score field can be parsed to double.
    */
-  private double parseScoreFromBytes(byte[] scoreInBytes) throws BitSailException {
+  private double parseScoreFromString(String scoreString) throws BitSailException {
```

Comment:
because when we get the value by record.getField(), from type converter we get String value, if use the old method, I need to call `parseScoreFromBytes(scoreOrHashKey.getBytes())`, I think maybe use `parseScoreFromString(scoreOrHashKey)` is more clearly, and avoid call getBytes method in this situation.